hs_core/hydroshare/users.py

```diff
@@ -645,23 +645,26 @@ def get_resource_list(
         if dc:
             for metadata in dc:
                 if metadata['content']:
+                    # TODO These are WAY too expensive to keep
                     queries[t] = filter(lambda r: r.dublin_metadata.filter(term=metadata['term']).exists(), queries[t])
                     queries[t] = filter(lambda r: r.dublin_metadata.filter(content=metadata['content']).exists(), queries[t])
+                    # queries[t] = queries[t].filter(dublin_metadata__term=metadata["term"])
```

Comment:
@pkdash as you pointed out, this is a left-over from dublin core cleanup and should be removed. Is it safe to just remove this "if dc" conditional block since I assume this condition will never be true now that dublin core dependency has been removed, right? If so, @ironfroggy could just remove this block as part of his filtering work.